src/Tasks/Microsoft.NET.Build.Tasks/Resources/Strings.resx

```diff
@@ -249,4 +249,10 @@
   <data name="RuntimeIdentifierWasNotSpecified" xml:space="preserve">
     <value>Specify a RuntimeIdentifier</value>
   </data>
+  <data name="AppHostHasBeenModified" xml:space="preserve">
+    <value>The App host has been modified, it is missing the default embedded value '{0}'</value>
```

Comment:
This will be very rare so I think the message should help identify the problem better.  Saying "modified" is a bit misleading as it could also be a bug where we get out of sync with apphost changes. This is a message that will be s

I suggest something like "Unable to use '{0}' as application host executable as it does not contain the expected placeholder byte sequence '{1}' that would mark where the application name would be written."

{0} would be the full path to the file so we can see package version, etc. if someone actually hits this.